### PR TITLE
allow teams to specify their organized editing policy url

### LINKED ISSUE #84: Add Organized editing policy to db

### PR DESCRIPTION
**Pull request by sethvincent**
closes #84 as long as we don't need to add a boolean or other field to express that a team has a policy that they are not yet linking to.

Screen Shot 2019-08-22 at 3 22 27 PM
Screen Shot 2019-08-22 at 3 22 16 PM

In addition to adding the organized editing policy column this pr improves the error handling in the TeamEditForm.

### CODE REVIEW

File: components/edit-team-form.js [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
 import { Formik, Field, Form } from 'formik'
+import urlRegex from 'url-regex'
 import Button from '../components/button'
 import dynamic from 'next/dynamic'
 const FormMap = dynamic(() => import('../components/form-map'), { ssr: false })
 
+const isUrl = urlRegex({ exact: true })
+
+function validateUrl (value) {
+  if (!isUrl.test(value)) return 'Please enter a valid url'
```
Comment (LanesGood) [inline components/edit-team-form.js]:
```suggestion
  if (value && !isUrl.test(value)) return 'Please enter a valid url'
```
Validation should only occur if user enters text into the input, allowing the URL to be optional.

File: components/edit-team-form.js [hunk 2]
```diff
@@ -1,9 +1,31 @@
 import React from 'react'
 import { Formik, Field, Form } from 'formik'
+import urlRegex from 'url-regex'
```
Comment (kamicut) [inline components/edit-team-form.js]: Should validation also happen at the API level?

File: components/edit-team-form.js (hunk 1 above)
Comment (sethvincent) [reply components/edit-team-form.js]: Yesss, thank you

File: components/edit-team-form.js (hunk 2 above)
Comment (sethvincent) [reply components/edit-team-form.js]: yeah it probably should

**Pull request merged**